README.adoc

```diff
@@ -0,0 +1,75 @@
+= Hazelcast Viridian Maven Plugin
+:description: The Hazelcast Viridian Maven Plugin is a Java development tool for testing and deploying link:https://docs.hazelcast.com/cloud/cluster-side-modules[cluster-side modules] on Serverless clusters.
+:page-plugin-version: 0.0.1
```

Comment:
I believe we should release version with build in Viridian URL first.

Reply:
Updated to 0.0.2